app/client/components/pages/singleBulkDisbursementPage.jsx

```diff
@@ -137,15 +139,15 @@ class SingleBulkDisbursementPage extends React.Component {
             <Space>
               <Button
                 onClick={() => this.onReject()}
-                disabled={status !== "PENDING"}
+                disabled={status == "APPROVED"}
```

Comment:
lol, being explicit makes more sense!

Reply:
It's to keep `PARTIAL` statuses from disabling the button

Reply:
yup